pkg/git/gitlab/gitlab_helper.go

```diff
@@ -91,6 +91,12 @@ func GetBaseUrl(repoUrl string) (string, error) {
 func refineGitHostingServiceError(response *http.Response, originErr error) error {
 	// go-gitlab APIs do not return a http.Response object if the error is not related to an HTTP request.
 	if response == nil {
+		if strings.Contains(originErr.Error(), "The provided authorization grant is invalid") || strings.Contains(originErr.Error(), "401 Unauthorized") {
+			return boerrors.NewBuildOpError(boerrors.EGitLabTokenUnauthorized, originErr)
+		}
+		if strings.Contains(originErr.Error(), "403 Forbidden") {
```

Comment:
I think we should distinguish between forbidden for specific resource and blocked account. It would make troubleshooting way easier.

Reply:
ok